thirdparty/assimp_fbx/FBXProperties.h

```diff
@@ -65,121 +67,130 @@ class Element;
 */
 class Property {
 protected:
-    Property();
+	Property();
```

Comment:
Are those indentation changes something made upstream which you synced from, or made by your IDE when doing your own modifications?

Reply:
When we move it to the new folder we can make it all godot formatting :)

Reply:
I don't remember off the top of my head, but I double checked it's correctly formatted.